modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -507,6 +507,10 @@ private String sanitizeKotlinSpecificNames(final String name) {
         return word;
     }
 
+    private boolean startWithDigits(String modified) {
```

Comment:
I don't think it's necessary to move such a simple regex to a function call. Also, the regex isn't digits (plural), it's a single digit.

Reply:
The regex is so simple that I make an error on. I preferred method even for an easy regex to avoid any effort when reading the code. But ok I will remove this.